kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PatternDetailsController.java

```diff
@@ -587,5 +630,9 @@ public void putTitlePanesArrowOnRight() {
         putArrowOnRight(this.fieldsTitledPane);
     }
 
-
+    @FXML
+    private void savePattern(ActionEvent actionEvent) {
+        boolean isValidSave = patternViewModel.createPattern();
```

Comment:
**Question:** after saving Do we care to know the PublicId? Do we need to display the identicon or the IDENTIFIER of the newly created Pattern. Also do we need to know the STAMP (used in the commit)?

Reply:
per @carldea  we will defer to other tickets